recipes/nghttp3/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import is_apple_os
+from conan.tools.cmake import CMake, CMakeToolchain, CMakeDeps, cmake_layout
```

Comment:
```suggestion
from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
```

CMakeDeps is not needed because does not require an external dependency.

Reply:
Got it. I'll remove CMakeDeps.